.github/workflows/__CD__deploy-image.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  workflow_call:
+    inputs:
+      app-name:
+        required: true
+        type: string
+    secrets:
+      ssh-host:
+        required: true
+      ssh-username:
+        required: true
+      ssh-priv-key:
+        required: true
+      secret-key-base:
+        required: true
+      phx-host:
+        required: true
+      ice-port-range:
+        required: true
+
+jobs:
+  deploy-image:
+    name: Deploy image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: ${{ inputs.app-name }}
+
```

Comment:
We don't really need that, do we? The tag is present in the `github` context, and the container image is pulled from the registry